src/goto-instrument/full_slicer.cpp

```diff
@@ -185,7 +185,8 @@ void full_slicert::add_jumps(
         if(cfg[entry->second].node_required)
         {
           const irep_idt id2=goto_programt::get_function_id(*d_it);
-          assert(id==id2);
+          INVARIANT(id==id2, "goto/jump expected to be "
+                    "within a single function");
```

Comment:
Nit picking: I'd break after the `,`

Reply:
OK. I'm happy to fix this, but there are already too many commits in this PR. It should be just one. What is the process for that? I cannot rebase, since the commits have already been pushed.

Reply:
Eventually you should squash all of them into one and force-push.